docs/howto/set-config.rst

```diff
@@ -28,7 +28,7 @@ can edit it to change any of the settings. You can also set options with environ
 For example, the
 metadata database connection string can either be set in ``airflow.cfg`` like this:
 
-.. code-block:: bash
+.. code-block:: ini
```

Comment:
```suggestion
.. code-block:: cfg
```
works, too - according to http://pygments.org/docs/lexers/#pygments.lexers.configs.IniLexer

Reply:
In other place, we use `ini`.  I prefer not to introduce many ways to mark the same. https://raw.githubusercontent.com/apache/airflow/dd9071bf5e127d449eee304736d2a5125813f607/docs/usage-cli.rst
This is only information about the file format and pygments select this format by calling this class "Ini*".

Reply:
👍